objects.go

```diff
@@ -824,17 +824,29 @@ func NewRecordTXT(
 }
 
 type ZoneAuth struct {
-	IBBase `json:"-"`
-	Ref    string `json:"_ref,omitempty"`
-	Fqdn   string `json:"fqdn,omitempty"`
-	View   string `json:"view,omitempty"`
-	Ea     EA     `json:"extattrs"`
+	IBBase          `json:"-"`
+	Ref             string `json:"_ref,omitempty"`
+	Fqdn            string `json:"fqdn,omitempty"`
+	View            string `json:"view,omitempty"`
+	NsGroup         string `json:"ns_group,omitempty"`
```

Comment:
What about setting no NS groups for this zone (in UI: 'None' for 'Name Servers')? 'omitempty' option can prevent this.

Reply:
If I understand correctly, you mean a case where the end user would like to send empty string "" as the `ns_group` to explicitly set this field to nothing; if we omit this from marshaling when it is empty, on the Infoblox side it will attempt to set the default NS Group instead. Is that correct?